### PR TITLE
pre-release version updates

### PR DESCRIPTION
**Pull request by koikonom**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,7 +47,7 @@ Before installing the chart, you must create two Kubernetes secrets:
 
 ```shell
 $ helm repo add hashicorp https://helm.releases.hashicorp.com
-$ helm search repo hashicorp/terraform --devel
+$ helm search repo hashicorp/terraform
 $ helm install --devel --namespace ${RELEASE_NAMESPACE} hashicorp/terraform --generate-name
```
Comment (redeux) [inline README.md:51]: Looks like the --devel flag is still referenced here and on new line 46 above.

**Comment by redeux** [comment]
Other than the --devel flag in 2 spots, LGTM!

**Comment by aareet** [comment]
@koikonom do the docs https://github.com/hashicorp/terraform-helm/blob/master/docs/workspace-sync.html.md also need an update with the new features? Last edit seems to be for TF_VERSION support

**Review by aareet — APPROVED** [review]

**Comment by aareet** [comment]
> @koikonom do the docs https://github.com/hashicorp/terraform-helm/blob/master/docs/workspace-sync.html.md also need an update with the new features? Last edit seems to be for TF_VERSION support

Non-blocking comment for this PR

**Comment by koikonom** [comment]
I found a last minute quoting issue in the deployment specification.

**Review by redeux — APPROVED** [review]
Blastoff!

**Pull request merged**